packages/sdk/client/src/services/shell-manager.ts

```diff
@@ -59,6 +59,11 @@ export class ShellManager {
     this._display = ShellDisplay.FULLSCREEN;
     this.contextUpdate.emit({ display: this._display });
     await this._shellRpc.rpc.ShellService.setLayout(request, { timeout: RPC_TIMEOUT });
+    (
```

Comment:
could you include a comment here explaining why this is needed/what it's intended to do?